plugins/org.jboss.tools.openshift.client/.classpath

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
-	<classpathentry exported="true" kind="lib" path="lib/openshift-restclient-java-4.0.3.Final.jar"/>
+	<classpathentry kind="lib" path="lib/openshift-restclient-java-4.0.3.Final.jar"/>
```

Comment:
shouldnt this be removed (we have a 2nd entry for the client lib 4.0.4 further down)